datapackage_pipelines_assembler/processors/validate_resource.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from datapackage import Package
+from goodtables import validate
+from datapackage_pipelines.wrapper import spew, ingest
+from datapackage_pipelines.utilities.resources import PROP_STREAMED_FROM
+
+parameters, datapackage, res_iter = ingest()
+report_name = 'datapackage_report.json'
+
+
+def generate_report(datapackage_):
+    reports = []
+    dp = Package(datapackage_)
+
+    for resource in dp.resources:
+        report = validate(
+            resource.descriptor[PROP_STREAMED_FROM],
```

Comment:
use the `path` attribute, not PROP_STREAMED_FROM

Reply:
@akariv I actually added this line cause I was getting `KeyError: 'dpp:streamedFrom'`